e2e-polybft/framework/test-server.go

```diff
@@ -263,3 +260,49 @@ func (t *TestServer) Withdraw(secrets string, recipient ethgo.Address) error {
 
 	return runCommand(t.clusterConfig.Binary, args, t.clusterConfig.GetStdout("withdrawal"))
 }
+
+func (t *TestServer) HasValidatorSealed(firstBlock, lastBlock uint64, validators polybft.AccountSet,
+	validatorAddr ethgo.Address) (bool, error) {
+	for i := firstBlock + 1; i <= lastBlock; i++ {
+		block, err := t.JSONRPC().Eth().GetBlockByNumber(ethgo.BlockNumber(i), false)
+		if err != nil {
+			return false, err
+		}
+
+		extra, err := polybft.GetIbftExtra(block.ExtraData)
+		if err != nil {
+			return false, err
+		}
+
+		validatorssignatures, _ := validators.GetFilteredValidators(extra.Parent.Bitmap)
```

Comment:
Add error checking here and also rename it to camel case (`validatorsSignatures`). Think you have added it previously, but when moved this function, it somehow got reverted.

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/2fc602b82356dcac8bcf92070e845db7b4e16ffe